content/developer/reference/frontend/assets.rst

```diff
@@ -38,191 +42,195 @@ a dedicated `assets` key which contains a dictionary. The dictionary will map
+files) and templates (`xml` files).
 
-  - the js files are concatenated and minified, then a script tag is generated
+Code
+    Odoo supports :ref:`three different kinds of javascript files<frontend/js_modules>`.
+    All these files are then processed (native JS modules are transformed into odoo
+    modules), then minified (if not in `debug=assets` mode) and concatenated. The
+    result is then saved as a file attachment. These file attachments are usually
+    loaded via a `<script>` tag in the `<head>` part of the page (as a static
+    file).
 
-.. note::
-   Assets files are cached, so in theory, a browser should only load them once.
+Style
+    Styling can be done with either `css` or `scss <https://sass-lang.com/>`_. Like
+    the javascript files, these files are processed (`scss` files are converted into
+    `css`), then minified (again, if not in `debug=assets` mode) and concatenated.
+    The result is then saved as a file attachment. They are then usually loaded
+    via a `<link>` tag in the `<head>` part of the page (as a static file).
 
-Main bundles
-------------
-When the Odoo server is started, it checks the timestamp of each file in a bundle
-and, if necessary, create/recreate the corresponding bundles.
+Template
+    Templates (static `xml` files) are handled in a different way: they are simply
+    read from the file system whenever they are needed, and concatenated.
 
-Here are some important bundles that most developers will need to know:
+    Whenever the browser loads odoo, it calls the `/web/webclient/qweb/` controller
+    to fetch the templates.
 
-- `web.assets_common`: this bundle contains most assets which are common to the
-  web client, the website, and also the point of sale. This is supposed to contain
-  lower level building blocks for the odoo framework. Note that it contains the
-  :file:`boot.js` file, which defines the odoo module system.
+It is useful to know that in most cases, a browser only performs a request the
+first time it loads a page. This is because each of these assets are
+associated with a checksum, which is injected into the page source. The checksum
+is then added to the url, which means that it is possible to safely set the cache
+headers to a long period.
 
-- `web.assets_backend`: this bundle contains the code specific to the web client
-  (notably the web client/action manager/views) and all static XML templates used
-  in the backend environment
+Operations on asset bundles
+===========================
 
-- `web.assets_frontend`: this bundle is about all that is specific to the public
-  website: ecommerce, forum, blog, event management, ...
+Typically, handling assets is simple: you just need to add some new files
+to a frequently used bundle like `common` or `backend`. But there are other operations
+available to cover some more specific use cases:
 
-Operations on asset bundles
----------------------------
+`append`
```

Comment:
Do you mind declaring a specific sub-section for each operation ?
There are 7 operations and a quick access on the right side without needing to scroll could be useful :).